contracts/Holdable.sol

```diff
@@ -109,23 +109,43 @@ contract Holdable is IHoldable, ERC20 {
         return true;
     }
 
-    function executeHold(string memory operationId, uint256 value) public returns (bool) {
+    function executeHold(string memory operationId, uint256 value) public returns (bool) { 
+        return _executeHold(operationId, value, false);
+    }
+
+    function executeHoldAndKeepOpen(string memory operationId, uint256 value) public returns (bool) {
```

Comment:
I'm not 100% sure about the name, mainly because we don't use `Open` as a status anywhere else. But I don't have a better suggestion and the name describes well what it does, so it's okay for me to keep it.